bootstrap-extensions/pom.xml

```diff
@@ -134,10 +139,6 @@
                     <groupId>org.webjars.bower</groupId>
                     <artifactId>bootstrap</artifactId>
                 </exclusion>
-                <exclusion>
```

Comment:
Are you sure about this ?
The managed dependency above (line 61) has different `groupId`. I have the feeling that now we may have two kind of dependencies to `font-awesome`.

Reply:
Well summernote-0.8.x is not pulling in a transitive dependency on font-awesome anymore, so there's not use excluding it.

Reply:
:+1: